website/docs/components/link/standalone/partials/guidelines/guidelines.md

```diff
@@ -3,105 +3,108 @@
 ### When to use
 
 - To navigate to a new destination, either internally or externally.
-- As a standalone element or within a Button Set, use a **(Link) Standalone**
+- As a standalone element or within a [Button Set](/components/button-set).
 
 ### When not to use
 
-- Within a body of text, use a **Link (Inline)**
-- To trigger an action or event, like a form submission, use a [Button](/components/button)
+- Within a body of text, consider [Inline Link](/components/link/inline).
+- To trigger an action or event, like a form submission, consider [Button](/components/button).
 
-### Size
+## Size
 
-<Hds::Link::Standalone @size="large" @text="Large" @icon="arrow-right" @iconPosition="trailing" @route="components" />
+We recommend using a medium size, but use whichever size fits best in the UI.
 
-<Hds::Link::Standalone @size="medium" @text="Medium" @icon="arrow-right" @iconPosition="trailing" @route="components" />
+For example:
 
-<Hds::Link::Standalone @size="small" @text="Small" @icon="arrow-right" @iconPosition="trailing" @route="components" />
+- don’t use large links in tables.
```

Comment:
Curious whether a bullet is needed here if it's only a single item. This is probably a relevant topic to capture holistically; a lot of the migration of design guidelines had lists with only a single item which seems strange.

Reply:
Good catch! I try to remove them if they seem randomly placed but have been leaving them in the "how to use" section (where a lot of them seem to be) since the "how not to use" section usually has multiple bullet points. My thought was to keep those two sections consistent and fix the rest. wdyt?